go/runtime/bundle/registry.go

```diff
@@ -147,11 +155,30 @@ func (r *Registry) Manifests() []*ExplodedManifest {
 	return slices.Collect(maps.Values(r.manifests))
 }
 
+// ManifestsWithLabels returns all manifests that have the specified labels set.
```

Comment:
Maybe mention that empty labels doesn't match any manifest (could also exit early in that case, potentially even error?)

Reply:
This is confusing. If this functions filters manifest with the given labels (and some additional labels as well), then the empty map should return all manifests.

Reply:
Fixed.